src/helpers/errors.js

```diff
@@ -60,9 +60,10 @@ export function printUnsafeWarning(functionName: string) {
   }
 
   console.warn(`
-  Deprecation Warning:
-  ${functionName} is not recommended for use and has been renamed to UNSAFE_${functionName}.
-  In react-native-testing-library 2.x only the UNSAFE_${functionName} name will work.
+    Deprecation Warning:
+    Warning:
+    ${functionName} promotes testing implementation details and is not recommended.
+    It may be removed in the future. Please test observable outcomes of rendering components.
```

Comment:
Do we still need to yell at users for using `UNSAFE_` functions now? I think the prefix is enough to realize there's something fishy about these APIs

Reply:
This is no longer called in UNSAFE_. We can remove this helper. What's your take?

Reply:
Removed remaining warnFn invocations in #334

Reply:
Let's do it

Reply:
I've removed the helper